### ISSUE TITLE
XML file metadata labels is incomplete

### ISSUE THREAD

**Issue by headdab** [opencv/cvat#16]
The labeling schema doesn't make it into the output XML file.

As an example, I created a job with a 'labels' spec of:

  person @select=type:white,blue,ref ball

and the dumped XML file is:

```
<?xml version="1.0" encoding="utf-8"?>                          
<annotations>                   
  <version>1.0</version>        
  <meta>                        
    <task>                      
      <id>16</id>               
      <name>test</name>         
      <size>902</size>          
      <mode>interpolation</mode>                                
      <overlap>5</overlap>      
      <bugtracker></bugtracker> 
      <created>2018-07-26 02:58:56.014598+03:00</created>       
      <updated>2018-07-26 02:58:56.014613+03:00</updated>       
      <labels>                  
        <label>                 
          <name>ball</name>     
          <attributes>          
          </attributes>         
        </label>                
      </labels>                 
      <segments>                
        <segment>               
          <id>24</id>           
          <start>0</start>      
          <stop>901</stop>      
          <url>http://13.66.164.80/?id=24</url>                 
        </segment>              
      </segments>               
      <owner>                   
        <username>cvat</username>                               
        <email>cvat@headdab.com</email>                         
      </owner>                  
    </task>                     
    <dumped>2018-07-26 02:59:11.669206+03:00</dumped>           
  </meta>                       
</annotations>         
```

         

Note that most of the 'labels' information is missing.  The only way I was able to be sure of my 'labels' spec for an existing job is that it was stored in the browser history.

Thanks.

**Comment by nmanovic** [comment]
Hi @headdab ,

Thank you for the report. It is a known problems which has been fixed internally. We will try to close the issue in the nearest future.

**Comment by nmanovic** [comment]
Hi @headdab 

It seems that the problem was fixed in 0.1.1. Could you please confirm? I cannot reproduce it with the version: https://github.com/opencv/cvat/commit/e8b2c4033022902a7be856583fe98b5fe7e0cb4b#diff-5005ece06f026f6e036eeba0602ddb00. I will close the issue. Don't hesitate to reopen if it still exists.

**Comment by headdab** [comment]
Looks good.
Thanks.
